webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js

```diff
@@ -189,6 +192,11 @@ const RegistrationCommandsPage = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [dispatch, hostGroupId, operatingSystemId]);
 
+  // Do not show the form if the user is not authorized to register hosts
+  if (apiDateErrorCode === 401 || apiDateErrorCode === 403) {
```

Comment:
HTTP 401 implies they're not logged in, so their session probably expired. Shouldn't they be shown the login page then?